src/BlockArea.cpp

```diff
@@ -441,10 +441,10 @@ bool cBlockArea::Read(cForEachChunkProvider * a_ForEachChunkProvider, int a_MinB
 		LOGWARNING("%s: MaxBlockY less than zero, adjusting to zero", __FUNCTION__);
 		a_MaxBlockY = 0;
 	}
-	else if (a_MaxBlockY > cChunkDef::Height)
+	else if (a_MaxBlockY >= cChunkDef::Height)
 	{
 		LOGWARNING("%s: MaxBlockY more than chunk height, adjusting to chunk height", __FUNCTION__);
-		a_MaxBlockY = cChunkDef::Height;
+		a_MaxBlockY = cChunkDef::Height - 1;
```

Comment:
Oh I see that's the goal of that block. Make them exclusive.

	// Include the Max coords:
	a_MaxBlockX += 1;
	a_MaxBlockY += 1;
	a_MaxBlockZ += 1;

My proposition doesn't make sense then, sorry.